app/views/lists/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Lists</h1>
+
+
+<% @lists.each do |list| %>
+<div>
+  <h2><%= list.title %></h2>
+  <p> <%= list.description %> </p>
+  <% list.books.each_with_index do |book, index| %>
```

Comment:
@ajkamel @evan-007 Here is one option if we don't want to have either N+1 or loading too much and filter in ruby side.

https://robots.thoughtbot.com/active-record-eager-loading-with-query-objects-and-decorators